support/ci/deploy_website.sh

```diff
@@ -7,16 +7,9 @@ if [[ "${TRAVIS_PULL_REQUEST}" = "false" ]] && [[ "${TRAVIS_BRANCH}" = "master"
   cd www
   make deploy
 else
-  if ! [[ "${TRAVIS_PULL_REQUEST}" = "false" ]] && [[ "${TRAVIS_BRANCH}" = "master" ]]; then
+  if [[ "${TRAVIS_PULL_REQUEST}" = "false" ]] && [[ "${TRAVIS_BRANCH}" =~ ^acceptance_deploy ]]; then
```

Comment:
nevermind. I don't think you do need that as long as the pr is targeting master. need coffee.